packages/graphql/src/index.ts

```diff
@@ -44,10 +46,10 @@ export const createGraphQLClient =
     headers?: RequestHeaders,
     fetchFn?: typeof fetch
   ): GraphQLClientQuery =>
-  (query, variables = {}, initialValue) =>
+  (query, variables, initialValue) =>
     createResource(
-      () => access(variables),
-      vars => {
+      () => access(variables == undefined ? {} : variables),
+      (vars: any) => {
```

Comment:
Are changes here necessary?
here `variables == undefined ? {} : variables` will be executed every time createResource calls the `source` function, although the `variables` argument once provided, cannot change, so there is no point checking if it's undefined multiple times.

Reply:
You're right, I just tried fixing type error that was caused by `variables = {}`